torch/csrc/distributed/rpc/script_call.cpp

```diff
@@ -58,9 +90,11 @@ std::shared_ptr<Operator> ScriptCall::fromIValues(
 
     ivalues.pop_back();
     // remove str_schema from ivalues
-    return op;
+    return std::make_unique<ScriptCall>(op, std::move(ivalues));
   } else {
-    TORCH_CHECK(false, "Unrecognized qualified name ", qualifiedName);
+    ivalues.pop_back();
```

Comment:
nit: ivalues.pop_back() can be outside the if-else block and be shared by both codepaths?